pkg/ast/ast.go

```diff
@@ -252,15 +252,36 @@ func (n *Int) GetName() string {
 	return n.Ident
 }
 
+type Operator int
+
+const (
+	_ Operator = iota
```

Comment:
You mean the `_` entry?
I think that by starting from 0 we risk treating a field that was initialized by default as the first enum evalue. Since we are here explicitly doing switches over operators, starting from 1 will help us detect more bugs in the code that deals with these operators.

I see that we can achieve the same effect by `= iota + 1`  though.

Reply:
Yes, `_` entry.
Let's do `= iota + 1`.